hw/ip/kmac/dv/kmac_base_sim_cfg.hjson

```diff
@@ -40,7 +40,7 @@
   // Default iterations for all tests - each test entry can override this.
   reseed: 50
 
-  run_opts: ["+test_timeout_ns=4000000000"]
+  run_opts: ["+test_timeout_ns=8000000000"]
```

Comment:
Perhaps the reader in $value$plusargs will accept underscores in numbers, so this could be written as 8_000_000_000 instead of 8000000000? Long numbers are just a pain, and with underscores this clearly means 8 seconds... which is a heck of a lot of simulation time. What takes so much time?

Reply:
good point - i can add underscores.

unfortunately i'm unable to see the actual log file for these timing out tests (as they're not available publicly), so my assumption is that it is due to a large number of iterations in the test sequence as well as a very large input size in many of these iterations.

However, I've  made many updates to the particular test that is timing out, so I will see regression results in the morning and re-assess the value of this PR.